base/linalg/lapack.jl

```diff
@@ -1205,8 +1205,8 @@ for (gelsd, gelsy, elty) in
             end
             newB = [B; zeros($elty, max(0, n - size(B, 1)), size(B, 2))]
             s     = similar(A, $elty, min(m, n))
-            rcond = convert($elty, rcond)
-            rnk   = Vector{BlasInt}(1)
+            rcond = Ref{$elty}(rcond)
+            rnk   = Ref{BlasInt}()
             info  = Ref{BlasInt}()
             work  = Vector{$elty}(1)
             lwork = BlasInt(-1)
```

Comment:
`&` is also used together with `m`, `n` and even expressions like `max(1, stride(A,2))`. Actually, could that be wrong for big-endian Int64 machines?